index.js

```diff
@@ -14,6 +14,8 @@ async function run() {
 		const exclude = core.getInput('exclude');
 		const dateFormat = core.getInput('date-format');
 		const reverseSort = core.getInput('reverse-sort');
+		const draftOption = core.getInput('draft') === 'true';
+		const prereleaseOption = core.getInput('prerelease') === 'true';
```

Comment:
```suggestion
		const isDraft = core.getInput('draft') === 'true';
		const isPrerelease = core.getInput('prerelease') === 'true';
```

And change the respective instances of these variables down below too.

Reply:
OK